dissect/target/plugins/os/windows/dpapi/keyprovider/lsa.py

```diff
@@ -49,7 +49,10 @@ def keys(self) -> Iterator[tuple[str, str]]:
             if default_pass := self.target.lsa._secrets.get(secret):
                 try:
                     value = c_defaultpassword.DefaultPassword(default_pass).data
-                except Exception:
-                    self.target.log.warning("Failed to parse LSA %s value", secret)
+                except Exception as e:
+                    self.target.log.warning(
+                        "Failed to parse LSA %s value (%r): %s", secret, default_pass.decode("utf-16-le"), e
```

Comment:
That decode seems sketchy, especially if we failed to parse it with the structure?